plans/provision/artemis/sanity.fmf

```diff
@@ -0,0 +1,28 @@
+#
+# Thanks to no "standard" public Artemis instance being available, Artemis
```

Comment:
we can make testing work via public artemis instance, do we want to?

Reply:
Which Artemis instance do you mean? If the one behind TFT, I'd argue that it's not really public, and I'd say "no" here.

Reply:
ok, it could be workarounded, but we can do that later, once auth is around.